packages/apps-config/src/api/spec/curio.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright 2017-2023 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { OverrideBundleDefinition } from '@polkadot/types/types';
+
+/* eslint-disable sort-keys */
+const definitions: OverrideBundleDefinition = {
+    types: [
```

Comment:
If you run `yarn lint` locally it should just (hopefully) fix the whitespace issues in one go. (Just an issue of using 4 spaces between the 2 as we generally use here)

Reply:
Done. Thank you